.reuse/dep5

```diff
@@ -10,16 +10,20 @@ Files: **/Makefile.am
        .github/*
        .gitignore
        Makefile.am
+       meson.build
+       meson_options.txt
        *.md
        configure.ac
-       configure.ac
        src/Makefile.am
+       src/meson.build
        src/provider.exports
```

Comment:
should we drop provider.exports here ?
Is it still used once we move to provider.map ?

Reply:
Removed.